numpy/distutils/system_info.py

```diff
@@ -1058,9 +1057,9 @@ def get_mkl_rootdir(self):
         for d in paths:
             dirs = glob(os.path.join(d, 'mkl', '*'))
             dirs += glob(os.path.join(d, 'mkl*'))
-            for d in dirs:
-                if os.path.isdir(os.path.join(d, 'lib')):
-                    return d
+            for dd in dirs:
+                if os.path.isdir(os.path.join(dd, 'lib')):
+                    return dd
```

Comment:
This is the only changeset in the PR that impacts actual code. The original code overrode `d` inside the loop, but the `mkl` code is probably never properly tested by us.

Reply:
Looks harmless, d was not used after the override anyway